addons/portal/views/portal_templates.xml

```diff
@@ -196,23 +196,40 @@
         </t>
     </template>
 
-    <template id="portal_docs_entry" name="My Portal Docs Entry">
-        <a t-att-href="url" t-att-title="title" class="list-group-item list-group-item-action d-flex align-items-center justify-content-between d-none">
-            <t t-esc="title"/>
-            <t t-if='count'>
-                <span class="badge text-bg-secondary rounded-pill" t-esc="count"/>
-            </t>
-            <t t-elif="placeholder_count">
-                <span class="badge text-bg-secondary rounded-pill" t-att-data-placeholder_count="placeholder_count">
-                    <i class="fa fa-spin fa-circle-o-notch"></i>
-                </span>
-            </t>
+    <template id="portal_docs_entry" name="My Portal Docs Entry">     
+    <div t-att-class="'col-md-6 ' + ('order-0' if bg_color else 'order-1')">
+        <a t-att-href="url" t-att-title="title" t-attf-class="card d-flex justify-content-center h-100 border-0 #{bg_color if bg_color else 'bg-light'}" style="max-width: 540px;">
+            <div t-attf-class="row g-0 #{'h-100' if bg_color else 'bg-light'}">
+                <div t-if="icon" class="col col-md-3 d-flex justify-content-center align-items-center">
+                    <img t-attf-src="#{icon}" width="50"/>
+                </div>
+                <div t-attf-class="#{'col-7' if icon else 'col-12'}">
+                    <div t-attf-class="card-body #{'d-flex align-items-center h-100' if bg_color else 'bg-light'}">
+                        <div t-attf-class="#{'d-flex gap-3' if placeholder_count or count else ''}">
+                            <t t-if='count'>
+                                <span class="badge text-bg-secondary rounded-pill" t-out="count"/>
+                            </t>
+                            <t t-elif="placeholder_count">
+                                <h5 class="mb-0" t-att-data-placeholder_count="placeholder_count">
+                                </h5>
+                            </t>
+                            <h5 class="card-title mb-0">
+                                <t t-out="title"/>
+                            </h5>
+                        </div>
+                        <p class="card-text">
+                            <t t-out="text"/>
+                        </p>
+                    </div>
+                </div>
+            </div>
         </a>
+    </div>
     </template>
 
     <template id="portal_table" name="My Portal Table">
-        <div t-attf-class="table-responsive border rounded border-top-0 #{classes if classes else ''}">
-            <table class="table rounded mb-0 bg-white o_portal_my_doc_table">
+        <div t-attf-class="table-responsive border-0 #{classes if classes else ''}">
+            <table class="o_list_table table table-sm table-hover position-relative mb-0 o_list_table_ungrouped table-striped o_portal_my_doc_table">
```

Comment:
UX is not good... table entries are highlighted on :hover, but actually you can click the title only. 
Either you remove the :hover effect or you make the entire row clickable ;) 

https://user-images.githubusercontent.com/8558351/234633983-e5059601-95c4-43a8-9877-a691e3ec6acf.mov